source/index.txt

```diff
@@ -9,8 +9,10 @@ Start Developing with MongoDB
 .. meta:: 
    :keywords: r, elixir, prisma, mongoose
    
-Connect your application to your database with one of the official MongoDB 
-libraries.
+Connect your application to your `MongoDB Atlas database
```

Comment:
Suggestion:
Similar to [your comment](https://github.com/mongodb/docs-node/pull/806#discussion_r1330481558) in a recent Node.js driver PR, I think "deployment" would be a more suitable term to use than "database".